app/pages/index.tsx

```diff
@@ -30,12 +29,11 @@ export default class Index extends Component<Props> {
   render() {
     const {query} = this.props;
     const {session} = query || {};
-    const date1: any = query.getReportingYear.applicationOpenDate;
-    const startDate: Date = new Date(Date.parse(date1));
-    const date2: any = query.getReportingYear.applicationEndDate;
-    const endDate: Date = new Date(Date.parse(date2));
-    const parsedStartDate = startDate.toDateString();
-    const parsedEndDate = endDate.toDateString();
+
+    const date1 = query.getReportingYear.applicationOpenDate;
+    const startDate = new Date(Date.parse(date1));
+    const date2 = query.getReportingYear.applicationEndDate;
+    const endDate = new Date(Date.parse(date2));
```

Comment:
Now the issue is that using `Date.parse` with a string is actually discouraged.
From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date:
> Note: Parsing of date strings with the Date constructor (and Date.parse(), which works the same way) is strongly discouraged due to browser differences and inconsistencies. Support for RFC 2822 format strings is by convention only. Support for ISO 8601 formats differs in that date-only strings (e.g. "1970-01-01") are treated as UTC, not local.

Looks like we should be using a library instead. I'm sure @wenzowski has an opinion on that :smile:  https://momentjs.com/ perhaps?